app/stacks/applications-service/variables.tf

```diff
@@ -15,6 +15,16 @@ variable "app_service_plan_id" {
   type        = string
 }
 
+variable "common_network_name" {
```

Comment:
Could we call this common_vnet_name? Just so it's clear it's a VNet resource and not something else?